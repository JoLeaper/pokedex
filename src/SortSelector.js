import React, { Component } from 'react';
import './SortSelector.css'

export default class SortSelector extends Component {
    render() {
        return (
            <div className="sort">
                <div className='sortOrder'>
                    <p>Display Order:</p>
                    <select value={this.props.displayOrder} onChange={this.props.handleOrderChange}>
                        <option value='asc'>Low to High</option>
                        <option value='desc'>High to Low</option>
                    </select>
                </div>
                <div className='sortBy'>
                    <p>Sort By:</p>
                    <select value={this.props.sortBy} onChange={this.props.handleSortChange}>
                        <option value='id'>ID</option>
                        <option value='attack'>Attack</option>
                        <option value='defense'>Defense</option>
                    </select>


                </div>

            </div>
        )
    }
}
