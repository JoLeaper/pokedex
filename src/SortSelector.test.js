import React from 'react';
import { shallow } from 'enzyme';
import SortSelector from './SortSelector'

test('renders sort selector', () => {
    const wrapper = shallow(<SortSelector />);
    expect(wrapper).toMatchSnapshot();
});
