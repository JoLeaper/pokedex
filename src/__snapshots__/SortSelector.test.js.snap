// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders sort selector 1`] = `
<div
  className="sort"
>
  <div
    className="sortOrder"
  >
    <p>
      Display Order:
    </p>
    <select>
      <option
        value="asc"
      >
        Low to High
      </option>
      <option
        value="desc"
      >
        High to Low
      </option>
    </select>
  </div>
  <div
    className="sortBy"
  >
    <p>
      Sort By:
    </p>
    <select>
      <option
        value="id"
      >
        ID
      </option>
      <option
        value="attack"
      >
        Attack
      </option>
      <option
        value="defense"
      >
        Defense
      </option>
    </select>
  </div>
</div>
`;
